spec/cobra_commander/executor_spec.rb

```diff
@@ -21,18 +21,9 @@
     expect(output.string).to match %r{===> node_manifest \(.*spec/fixtures/app/node_manifest\)}
   end
 
-  it "executes with clean environment" do
-    output = StringIO.new
-    CobraCommander::Executor.exec(umbrella.components, "env", output)
+  it "cleans BUNDLE environment so a nested bundle call can succeed" do
+    expect(Bundler).to receive(:with_original_env).and_yield
```

Comment:
This isn't very valuable. Can we assert behaviour instead?

Reply:
I'll think of a way of doing this

Reply:
I'm trying to think of something valid to assert that the original bundler env var's are there but I can't think of anything that doesn't end-up testing exactly what `with_original_env` is designed for.